assignment/scripts/3-music-collection.js

```diff
@@ -1 +1,180 @@
 console.log('***** Music Collection *****')
+
+let Collection = [];
```

Comment:
Typically in JavaScript you would use _lower_ camel case to write this array name.

```suggestion
let collection = [];
```